### ISSUE TITLE
aha-table handleValidateEvent

### ISSUE THREAD

**Issue by micahriggan** [predixdesignsystem/px-data-table#110]
https://github.com/PredixDev/px-data-table/blob/master/aha-table.html#L498

It appears evt is being referenced, but I can't seem to find a variable named that in scope.

Also I believe that _readContent should only be called when column isn't null.

```
if (column != null && this._cellChanged(this._readContent(this._internalData[index], column), changeRec.value)) {                                                                                 
              this._handleValidateEvent(evt);                                                                                                                                                               
          }
```

**Comment by randyaskin** [comment]
See also #77

**Comment by randyaskin** [comment]
I believe I fixed the handleValidate part in 2.0.0 - need to look again at your PR for the other part.

**Comment by mdwragg** [comment]
Closing due to age and release of https://www.predix-ui.com/#/elements/data-grid/px-data-grid
